modules/dynamic-plugins/proc-topology-configure.adoc

```diff
@@ -0,0 +1,326 @@
+= Configuration
+
+== Viewing OpenShift routes
+To view OpenShift routes, you must grant read access to the routes resource in the Cluster Role:
+
+[source,yaml]
+----
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: backstage-read-only
+  rules:
+    ...
+    - apiGroups:
+        - route.openshift.io
+      resources:
+        - routes
+      verbs:
+        - get
+        - list
+----
+
+You must also add the following in kubernetes.customResources property in your `app-config.yaml` file:
+
+[source,yaml]
+----
+kubernetes:
+    ...
+    customResources:
+      - group: 'route.openshift.io'
+        apiVersion: 'v1'
+        	  plural: 'routes'
+----
+
+== Viewing pod logs
+To view pod logs, you must grant the following permission to the ClusterRole: 
+
+[source,yaml]
+----
+ apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: backstage-read-only
+  rules:
+    ...
+    - apiGroups:
+        - ''
+      resources:
+        - pods
+        - pods/log
+      verbs:
+        - get
+        - list
+        - watch
+----
+
+== Viewing Tekton PipelineRuns
+To view the Tekton PipelineRuns you must grant read access to the Pipelines, PipelineRuns, and TaskRuns resources in the ClusterRole:
```

Comment:
Should "PipeLines" be `pipelines`, "PipeLineRuns"  be `pipelineruns` , and "TaskRuns" be `taskruns`?

`ClusterRole`

Reply:
Yes I thinks so.